GitUI/BuildServerIntegration/BuildServerWatcher.cs

```diff
@@ -161,10 +163,22 @@ public void CancelBuildStatusFetchOperation()
 
                             if (section is not null)
                             {
-                                buildServerCredentials.UseGuestAccess = section.GetValueAsBool(UseGuestAccessKey,
-                                    true);
+                                string? buildServerCredentialsType = section.GetValue(BuildServerCredentialsTypeKey);
+                                if (!string.IsNullOrWhiteSpace(buildServerCredentialsType))
+                                {
+                                    buildServerCredentials.BuildServerCredentialsType = Enum.Parse<BuildServerCredentialsType>(buildServerCredentialsType);
```

Comment:
What would happen if `buildServerCredentialsType = 143`? That is, a value is not defined.

Reply:
Good point - it would crash :)
![image](https://user-images.githubusercontent.com/483659/197138900-d4c1a69f-fff4-4eb1-9177-23686e0119ca.png)

I'll change it to `TryParse` instead
![image](https://user-images.githubusercontent.com/483659/197138980-7134b8ca-dbe1-44dc-a64a-58ba3b153ee9.png)